bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -352,7 +352,7 @@ static cl::opt<bool> CreateDebugNames(
 
 static cl::opt<bool>
     DebugSkeletonCu("debug-skeleton-cu",
-                    cl::desc("prints out offsetrs for abbrev and debu_info of "
+                    cl::desc("prints out offsets for abbrev and debug_info of "
```

Comment:
nit: it's better to put this change under a separate "[NFC]" commit.

Reply:
thanks! done in [PR#93955](https://github.com/llvm/llvm-project/pull/93955)